src/Components/Tab/component.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import { Tab as Base } from 'react-bootstrap';
+
+type TabProps = {
```

Comment:
if those are not used in the exactly this file, then there is no need to describe those properties

Reply:
This declaration serves as a model for the component props.

Reply:
I can see only one https://react-bootstrap.netlify.com/components/tabs/#tab-props prop here.